js/languages/en-US.json

```diff
@@ -1088,6 +1088,7 @@
     "quantity": "Quantity",
     "useShapeShift": "Or use ShapeShift in the next step",
     "moderatedPayment": "Moderated Payment",
+    "tooLowForMods": "Based on the current estimated fee, moderated payment is not available for orders under %{minModPrice}. A direct payment will be used instead.",
```

Comment:
It's singular because it's referring to the control, which is labeled "Moderated Payment", not the general idea of moderated payments. It could be made more clear though.